packages/flutter/lib/src/material/app.dart

```diff
@@ -218,164 +218,48 @@ class MaterialApp extends StatefulWidget {
   /// [builder] must not be null.
   final Map<String, WidgetBuilder> routes;
```

Comment:
iirc, some of the docs in this file were subtly different than those in widgets/app.dart in various minor ways. Please verify that the ones you've removed are in fact the same byte-for-byte.

Reply:
Reverted some unintended merging of MaterialApp's doc into WidgetsApp. Mainly references to `routes`. Double checked that the delete docs from MaterialApp appear word for word in the template.